logger.ts

```diff
@@ -0,0 +1,39 @@
+import { createLogger, format, transports } from 'winston';
+
+const { combine, timestamp, colorize, printf } = format;
+
+const defaultFormatter = printf(({ level, message, timestamp }) => `${timestamp} [${level}] : ${message}`);
+
+export const logger = createLogger({
+  level: 'info',
```

Comment:
During development, always set logger to `debug`, while in production set it to `error`
```suggestion
  level: 'debug',
```

It's a good idea to add LOG_LEVEL to the .env so that and control it from there. Something like this.

```
const level = process.env.LOG_LEVEL || 'info';
```